nevergrad/functions/ml/mlfunctionlib.py

```diff
@@ -28,7 +29,7 @@ class MLTuning(ExperimentFunction):
     # Example of ML problem.
     def _ml_parametrization(self,
                             depth: int,  # Parameters for regression trees.
-                            dimension: int,
+                            data_dimension: int,
```

Comment:
why not dimension? to differentiate from the dimension of the optimization?
maybe data_size is shorter and makes the point that it's different

Reply:
Yes, for that reason.
data_size is ambiguous to me, whereas data_dimension looks good enough.